ovirt_imageio/client/_options.py

```diff
@@ -45,6 +45,10 @@ def __repr__(self):
 log_level = Choices("log_level", ("debug", "info", "warning", "error"))
 
 
+def strtobool(str):
+    return str in ['True', 'true', '1']
```

Comment:
In conf file "Yes" and "yes" are also considered True. Check how configparser getbool() works. Can we use it to extract the value based on option.action?

Reply:
Just for the record, `getboolean` is implemented as a dictionary for possible states:
```python
    # Possible boolean values in the configuration.
    BOOLEAN_STATES = {'1': True, 'yes': True, 'true': True, 'on': True,
                      '0': False, 'no': False, 'false': False, 'off': False}
```
OK, I'll try to use it instead of setting the `type`.

Reply:
Using getboolean() is probably best, but I don't think it works with unset key.
Using get() we can detect if we have a value, and then convert the value to 
bool using the same rules (maybe using the dict you show here).